sendgrid_backend/mail.py

```diff
@@ -240,29 +240,28 @@ def _parse_email_address(self, address: str) -> Tuple[str, Optional[str]]:
             name = None
         return addr, name
 
-    def _build_sg_mail(self, msg: EmailMessage) -> Dict:
-        """
-        Serializes a Django EmailMessage into its JSON representation.
-
-        Returns a Dict of mail data to be consumed by the sendgrid api.
-        """
-        mail = Mail()
-
-        mail.from_email = Email(*self._parse_email_address(msg.from_email))
-
+    def _build_sg_personalization(
+        self,
+        to: Iterable[str],
+        msg: EmailMessage,
+        extra_headers: Iterable[Header],
+        personalizations: Dict = None,
+    ) -> Personalization:
+
```

Comment:
Can you please add a docstring to this method describing what is expected from each arg, and that the `personalizations` dict takes precedence over `msg`?

Reply:
Wasn't sure what style you wanted so I just went detail oriented